masu/celery/tasks.py

```diff
@@ -0,0 +1,111 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Celery task definitions."""
+
+# pylint: disable=unused-argument, fixme, unused-import
+# FIXME: temporary module-wide disable until tasks are fully implemented.
+
+from unittest import mock
+
+from celery.utils.log import get_task_logger
+
+from masu import create_app
+from masu.celery import create_celery
+from masu.processor.exceptions import MasuProcessingError, MasuProviderError
+
+MASU = create_app()
```

Comment:
I think I probably led you astray of the @shared_task the other day, but I think we can ultimately use them here so we don't have to create an app or celery instance. the autodiscovery process in the app factory will then find all of these and associate them with the celery instance we make there. We can experiment with that later when we build these tasks out further.